vnext/Microsoft.ReactNative/Fabric/Composition/CompositionDynamicAutomationProvider.cpp

```diff
@@ -247,7 +247,8 @@ HRESULT __stdcall CompositionDynamicAutomationProvider::GetPropertyValue(PROPERT
     }
     case UIA_NamePropertyId: {
       pRetVal->vt = VT_BSTR;
-      auto wideName = ::Microsoft::Common::Unicode::Utf8ToUtf16(props->accessibilityLabel);
+      auto wideName = ::Microsoft::Common::Unicode::Utf8ToUtf16(
```

Comment:
This would avoid the extra prop access and Utf8ToUtf16 on empty strings:
`auto wideName = props->accessibilityLabel.empty() ? ::Microsoft::Common::Unicode::Utf8ToUtf16(baseView->DefaultAccessibleName()) : L"";`

Reply:
@chrisglein You're logic looks fine.

@acoates-ms The logic in your example isn't equivalent as it would never use the `accessibilityLabel` value. If the label has a value, you want to use it, not report an empty string.

Reply:
Yeah, sorry... read it wrong.